src/ResourceManager/Network/Commands.Network/VirtualNetworkGatewayConnection/NewAzureVirtualNetworkGatewayConnectionCommand.cs

```diff
@@ -172,22 +172,14 @@ private PSVirtualNetworkGatewayConnection CreateVirtualNetworkGatewayConnection(
             vnetGatewayConnection.ConnectionType = this.ConnectionType;
             vnetGatewayConnection.RoutingWeight = this.RoutingWeight;
             vnetGatewayConnection.SharedKey = this.SharedKey;
-            
-            if (!string.IsNullOrEmpty(this.EnableBgp))
-            {
-                vnetGatewayConnection.EnableBgp = bool.Parse(this.EnableBgp);
-            }
-            else
-            {
-                vnetGatewayConnection.EnableBgp = false;
-            }
+
+            vnetGatewayConnection.EnableBgp = this.EnableBgp;
```

Comment:
Now you are stting a value regardless of whether the user set this value, which is different than before, where you only set the value if the user provided it.  Is this intentional?

Note that you can get the same behavior usign bool parameters if you use MyInvocation.BoundParameters to see if the user provided a value